plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraSession.java

```diff
@@ -571,7 +571,7 @@ private <T> T executeWithSession(SessionCallable<T> sessionCallable)
                     throw e;
                 }
                 long delay = Math.min(schedule.nextDelay().toMillis(), timeLeft);
-                log.warn(e.getMessage());
```

Comment:
does `log.warn(String)` do any interpolation?

Reply:
> does `log.warn(String)` do any interpolation?

Not at this moment. I made a change upstream to make it easier to detect places like this, but it's a semantics change.

Reply:
we should not do a semantics change.
what's the problem with `log.warn(String)` not doing any interpolation?

Reply:
> what's the problem with `log.warn(String)` not doing any interpolation?

It's arguably inconsistent with the overload which does take parameters. If this overload didn't exist, or if it would be marked as a format method, it would behave the same way the `String.format("abcde")` behaves, which does do interpolation (the only interpolation it can do is `"%%"`, though). But this is very arguable and people's expectations differ. I would expect this to be consistent across all invocations, but other people don't like it.

But the primary reason I would like `log.warn(String)` to be a format method is to allow Error Prone detect cases where the log message is constructed using concatenation. `log.warn(e.getMessage());` is a casualty of this, unfortunately :)